release-notes/20220803-subctl-directory.md

```diff
@@ -0,0 +1,3 @@
+<!-- markdownlint-disable MD041 -->
+If it’s not given a custom directory, `subctl gather` stores all its output in a directory
+named `submariner-` followed by the current date and time (in UTC) in "YYYYMMDDHHmmss" format.
```

Comment:
https://github.com/submariner-io/subctl/pull/136 creates subdirectories for each cluster.

```suggestion
The `subctl gather` command now creates a subdirectory for each cluster instead of embedding the cluster name in the file names.
```